src/util/fs_utils.js

```diff
@@ -142,8 +142,8 @@ function try_read_file_sync(file_name) {
     try {
         return fs.readFileSync(file_name, 'utf8');
     } catch (err) {
-        if (err.code === 'ENOENT' || err.code === 'ENOTDIR') {
-            // file does not exist or is not a directory
+        if (err.code === 'ENOENT' || err.code === 'EISDIR') {
+            // file does not exist or is a directory
```

Comment:
Hi @shylesh, Thanks for your PR. You correctly identified the wrong error code. In the context of this function, we should only ignore `ENOENT`, the second error code was put by mistake. If trying to read from a directory, the error should be thrown. 

```suggestion
        if (err.code === 'ENOENT') {
            // file does not exist 
```

Reply:
@dannyzaken ack, I will update.

Reply:
done